Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/combat_correspondent.yml

```diff
@@ -17,55 +19,81 @@
       takeAll: CMStandard
       entries:
       - id: RMCVendorBundleCrewCombatCorrespondent
-      # Portable Press Fax Machine - /obj/item/device/fax_backpack
-      - id: RMCCameraBroadcasting
+        name: Essential Reporter's Set
+        mandatory: true
       - id: CMSatchel
+        name: Leather Satchel
+      - id: RMCCameraBroadcasting
+        name: Press Broadcasting Camera
+        recommended: true
+    #- name: Civilian Equipment TODO - Some way to ensure only the civilian correspondent can buy this
+    #  takeAll: CMStandard
+    #  entries:
+    #  - id: PortableFaxMachine
+    #    name: Portable Press Fax Machine
+    #    recommended: true
     - name: Uniform
       choices: { CMUniform: 1 }
       entries:
       - id: RMCJumpsuitCorrespondentBlack
-        name: black uniform
+        name: Black Uniform
       - id: RMCJumpsuitCorrespondentOrange
-        name: orange uniform
+        name: Orange Uniform
       - id: RMCJumpsuitCorrespondentRed
-        name: red uniform
+        name: Red Uniform
     - name: Armor
       choices: { CMArmor: 1 }
       entries:
       - id: CMArmorPress
+        name: Combat Correspondent's Armor
```

Comment:
For Combat Correspondent we are waiting until that is its own role to hand out their gear, this should only be civilian correspondent gear for now unless that is named the same for some reason